src/components/Navbar.js

```diff
@@ -45,7 +46,7 @@ export default function Navbar() {
       <div className="navbar bg-base-100">
         <div className="flex-1">
           <div className="flex flex-row">
-            <Image src={Logo} alt="RepoRater Logo" width={40} height={40} />
+            <Image src={Logo} alt="RepoRater Logo" width={40} height={40} className="cursor-pointer" onClick={() => router.push("/")} />
```

Comment:
What about wrapping it in a `Link` component?

Reply:
sure, i will update the pull request with the requested changes.

Reply:
Thanks 👍